Gems/ROS2/Code/Source/RobotImporter/URDF/URDFPrefabMaker.cpp

```diff
@@ -0,0 +1,332 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "RobotImporter/URDF/URDFPrefabMaker.h"
+#include "ROS2/Frame/ROS2FrameComponent.h"
+#include "ROS2/ROS2GemUtilities.h"
+#include "ROS2/Spawner/SpawnerBus.h"
+#include "RobotControl/ROS2RobotControlComponent.h"
+#include "RobotImporter/URDF/CollidersMaker.h"
+#include "RobotImporter/URDF/PrefabMakerUtils.h"
+#include "RobotImporter/Utils/RobotImporterUtils.h"
```

Comment:
Well these includes could be adjusted.
```suggestion

#include "URDFPrefabMaker.h"
#include <ROS2/Frame/ROS2FrameComponent.h>
#include <ROS2/ROS2GemUtilities.h>
#include <ROS2/Spawner/SpawnerBus.h>
#include <RobotControl/ROS2RobotControlComponent.h>
#include "CollidersMaker.h"
#include "PrefabMakerUtils.h"
#include <RobotImporter/Utils/RobotImporterUtils.h>
```